k9mail/src/main/java/security/planck/ui/common/compose/input/PasswordInputField.kt

```diff
@@ -33,19 +33,19 @@ fun PasswordInputField(
     textColor: Color,
     errorColor: Color,
     defaultColor: Color,
-    evaluateError: (TextFieldStateContract) -> Unit,
+    onTextChanged: (String) -> Unit,
     modifier: Modifier = Modifier,
 ) {
     var passwordVisible by remember { mutableStateOf(false) }
 
     OutlinedTextField(
-        value = passwordState.textState,
+        value = passwordState.text,
         onValueChange = {
-            passwordState.textState = it
-            evaluateError(passwordState)
+            Log.e("EFA-602", "MyScreen text pawwword changed: ${passwordState.text}")
```

Comment:
typo I guess, even if it is in the logs should be fixed

Reply:
logs should be removed, thanks.